vllm/model_executor/models/interfaces.py

```diff
@@ -130,6 +131,62 @@ def supports_multimodal(
     return isinstance(model, SupportsMultiModal)
 
 
+@runtime_checkable
+class SupportsScoreTemplate(Protocol):
+    """The interface required for all models that support score template."""
+
+    supports_score_template: ClassVar[Literal[True]] = True
+    """
+    A flag that indicates this model supports score template.
+
+    Note:
+        There is no need to redefine this flag if this class is in the
+        MRO of your model class.
+    """
+
+    @classmethod
+    def get_score_template(cls, query: str, document: str) -> Optional[str]:
+        """
+        Generate a full prompt by populating the score template with query and document content.
+        """
+        ...
+
+    @classmethod
+    def post_process_tokens(cls, prompt: TokensPrompt):
```

Comment:
```suggestion
    def post_process_tokens(cls, prompt: TokensPrompt) -> None:
```

Better explicitly define the return type in interfaces